### ISSUE TITLE
EnderIO Crash/Conflict

### ISSUE THREAD

**Issue by MokahTGS** [MinecraftModDevelopmentMods/MMDLib#45]
Getting a crash when this is combined with EnderIO.  Any ideas?

Crash Log: https://pastebin.com/EKG2wUFV

**Comment by dshadowwolf** [comment]
``[14:09:05] [Client thread/ERROR]: Duplicate recipe 'mmdlib: ingotEmpty to dustEmpty'. A recipe with the same name was already read from IMC from mod 'mmdlib' at [row,col]:[2,65]``

This is looking like something went sideways in generating the recipes - basically... it looks like its trying to iterate the "Empty Material" and falling flat. This is a strange one, I'll have to go look, as MMDLib should not be registering anything itself...

**Comment by MokahTGS** [comment]
It has something to do with the integration because turning off the option for EnderIO integration in the configs stopped the crash.

**Comment by dshadowwolf** [comment]
yes - I might not have said it, but something is iterating the "Empty Material" - ie: the thing that exists for error checking and consistency purposes in MMDLib - and trying to gen recipes for it... That means something has gone completely sideways somewhere

**Comment by SuperFanatic** [comment]
I have this exact same problem. Can't install Ender IO because it gives me the "ingotEmpty to dustEmpty" crash.
What do you mean "turning off the option for EnderIO integration in the configs" I can't find an option for Ender IO int in the MMDLib config.

**Comment by dshadowwolf** [comment]
This is a bork in the EIO interface code where it isn't ignoring an internal book-keeping metric when trying to register stuff with the SAG Mill. Will be fixed in the next release. Sorry for the issue.
